test/files/neg/divergent-implicit.check

```diff
@@ -3,7 +3,7 @@ divergent-implicit.scala:4: error: type mismatch;
  required: String
   val x1: String = 1
                    ^
-divergent-implicit.scala:5: error: diverging implicit expansion for type Int => String
+divergent-implicit.scala:5: error: diverging implicit expansion for type Int => Nothing
```

Comment:
That's the only error message that got worse. Well I guess there is no complete happiness.

Reply:
But it's also nothing to be sad about.

Reply:
At that point the user is already very sad.

Reply:
You need a history of messages, and then pick the most specific, with an option to show all.